RSA/simple2.c

```diff
@@ -0,0 +1,192 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+#include <math.h>
+
+//функция, котоая проверяет число на простоту
+int isPrime(long int n){
+        for (long int i = 2; i*i <= n; ++i){
+                if (n % i == 0) {
+                        return 0;
+                }
+        }
+        return 1;
+}
+
+//функция, которая возвращает простое рандомное число длины,
+//введенной с клавиатуры и больше
+//upd: длина простого числа определяется рандомно
+long int lookingForSimple() {
+
+        int n;
+	//srand(time(NULL));
+	n = rand()%7;
+        //scanf("%d", &n);
+
+        long int num = 1;
+        for (int i = 1; i <= n; i++){
+                srand(time(NULL));
```

Comment:
No reason to do `srand(time(NULL))` every time in a loop. It should be done once in the start of the program. Call for `srand` everytime worsens perfomance.